tests/core/editable/keystrokes/delbackspacequirks/manual/noselection.md

```diff
@@ -0,0 +1,14 @@
+@bender-tags: 4.7.0, tc, 13096
+@bender-ui: collapsed
+@bender-ckeditor-plugins: toolbar, wysiwygarea
+
+1. Open browser console.
```

Comment:
Hm, restricting manual test to only one browser doesn't seem like good solution. Maybe trying to somehow change the test would be better? E.g. add button that removes selection from the editor?